src/complex/Filter/Parameters.cpp

```diff
@@ -49,4 +51,21 @@ void Parameters::insert(IParameter::UniquePointer parameter)
   std::string name = parameter->name();
   m_Params.insert({std::move(name), std::move(parameter)});
 }
+
+std::string Parameters::getAcceptedKeys() const
```

Comment:
Instead of `std::string getAcceptedKeys() const`, maybe we could have `std::vector<std::string> getKeys() const`. That could be reused in more places. Then in `IFilter` we can use `fmt::join(params.getKeys(), ", ")` instead.

Reply:
Done. Returns std::vector<std::string>